src/test/scala/com/cloudera/sparkts/models/ARIMAXSuite.scala

```diff
@@ -54,18 +55,372 @@ class ARIMAXSuite extends FunSuite {
     24,24,25,25,25,25,25,25,25,23,23,23,23,23,23,23,
     51,54,49,46,42,41,45,46,48,41,42,48,43,47,48,46 ))
 
+  def average(v: DenseVector): Double = {
+    var sum = 0.0
+    v.values.map( va => sum += va)
+    sum/tsTrain.values.length
+  }
+  def mse(orginal: BreezeDenseVector[Double], predicted: BreezeDenseVector[Double]): Double = {
+    var sumSquaredError = 0.0
+    for (o <- orginal; p <- predicted){
+      val error = o - p
+      sumSquaredError += Math.pow(error, 2)
+    }
+    val mse = Math.sqrt(sumSquaredError / orginal.length)
+    println(s"MSE =$mse")
+    mse
+  }
+
+  test("1 MAX(0,0,1) 1 t"){
+    val a: Double = average(tsTrain)
+    val model1 = ARIMAX.fitModel(0, 0, 1, tsTrain, xregTrain, 1, true)
+    assert( model1.coefficients.length == 6)
+    val results = model1.predict(tsTest, xregTest)
+    results.length should be (tsTest.length)
+    results.map( v => v should be (a +- 10))
```

Comment:
In this test, you're checking the results values, which is good, but I noticed in other tests you aren't.  Do those other parameter combinations not return reasonable results that we can compare?